packages/lab/src/components/Flow/DroppableFlow.tsx

```diff
@@ -144,6 +145,10 @@ export const HvDroppableFlow = ({
     id: elementId,
   });
 
+  useEffect(() => {
```

Comment:
It doesn't seem to be working when testing the "Initial state" sample. The `ref` is set before `onNodesChange` is triggered. 

I believe one way of solving this is using the [`onInit`](https://reactflow.dev/api-reference/react-flow#on-init) callback:

```
const handleInit = useCallback(= (i) => {
   initialRender.current = false;
   onInitProp?.(i) // Extracted from props passed by the used
}, [onInitProp])

<ReactFlow
   (...)
   onInit={handleInit}
>
{children}
</ReactFlow>
```